packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -2554,6 +2563,19 @@ class RawDialogRoute<T> extends PopupRoute<T> {
   /// {@macro flutter.widgets.DisplayFeatureSubScreen.anchorPoint}
   final Offset? anchorPoint;
 
+  /// {@template flutter.widgets.RawDialogRoute.fullscreenDialog}
```

Comment:
I believe this template is no longer needed since it's only used once.

Reply:
Reminder that this comment is still effective. :)

Reply:
@dkwingsmt We are using same template for `fullscreenDialog` of ModalRoute, Should template be there and this place we consume that ?

Reply:
Oh I see. Sorry I missed it!